test/safe/push.test.ts

```diff
@@ -390,4 +465,134 @@ describe("GitHub Actions", () => {
 			await expect(processPushJob(app)(job, getLogger('test'))).toResolve();
 		});
 	});
+
+	describe("end 2 end tests with queue", () => {
```

Comment:
That's the end to end test involving queue.

For SQS pullrequest https://github.com/atlassian/github-for-jira/pull/884 I'll add more test cases with sqs feature flag on and off